data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	swizzle 6
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25100000
+		"shields" 37200
+		"hull" 11780
+		"required crew" 62
+		"bunks" 70
+		"mass" 725
+		"drag" 9.5
+		"heat dissipation" .4
+		"fuel capacity" 500
+		"outfit space" 872
+		"weapon capacity" 333
+		"engine capacity" 150
+		weapon
+			"blast radius" 500
+			"shield damage" 5000
+			"hull damage" 2500
+			"hit force" 3500
+	outfits
+		"Large Heat Shunt" 2
+		"Jump Drive"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Quantum Keystone"
+		"Geode Reactor"
+		"White Sun Reactor"
+		`"Bufaer" Atomic Steering`
+		Sunbeam 4
+		"Thunderhead Launcher" 4
+		"Thunderhead Missile" 160
+		"Bullfrog Anti-Missile" 2
+		"Korath Warder"
+		"Thruster (Planetary Class)"
+		"Fuel Processor" 2
+		"Wanderer Ramscoop"
+		"Pulse Rifle" 69
+		"Blade Of Might"
+	engine -45 123
+	engine 45 123
+	gun -22 -103 Sunbeam
+	gun 22 -103 Sunbeam
+	gun -34.5 -68
+	gun 34.5 -68
+	gun -48 -53.5 Sunbeam
+	gun 48 -53.5 Sunbeam
+	gun -61.5 -23
+	gun 61.5 -23
+	turret 0 5 "Korath Warder"
+	turret -55.5 74.5
+	turret 55.5 74.5
+	leak "leak" 35 45
+	leak "flame" 60 95
+	leak "big leak" 50 45
+	explode "tiny explosion" 60
+	explode "small explosion" 51
+	explode "medium explosion" 54
+	explode "large explosion" 53
+	explode "huge explosion" 22
+	"final explode" "final explosion large"
+	description "This ship probably used to be a Shield Beetle, but it has received many modifications, with the obvious intent of survivability, including an enhanced shield matrix. One thing is certain, it is no longer simply meant to escort cargo, this is a proper ship for a warlord."
```

Comment:
The ship should have cargo space. Even an ambassadorial ship needs to carry supplies and mission cargo.

Reply:
It sometimes does, and when it does they either have an escort, or they modify the ship
The outfits expansion shouldnt be uses in these designs

Reply:
The Korath automata ships have tiny cargo holds that fit 1-2 outfits expansions since the ship is mainly intended to fight. You can still add the outfits expansions, but it's silly to do so. I'd like to model it off of that. Perhaps 33 or 47 or thereabouts. I'll experiment and see what works well.